index.js

```diff
@@ -28,6 +28,18 @@ MongoClient.connect('mongodb://localhost/sentiment', (err, db) => {
       })
   })
 
+  app.get('/api/terms/:term', (req, res) => {
```

Comment:
If you want all your routes to be on `/api` you could create a Router and mount it on `/api`.